packages/geo/src/lib/geometry/geometry-form-field/geometry-form-field-input.component.ts

```diff
@@ -200,6 +230,7 @@ export class GeometryFormFieldInputComponent implements OnInit, OnDestroy, Contr
     this.ready = false;
 
     this.deactivateControl();
+    this.cdRef.detach();
```

Comment:
Why is this line needed?

Reply:
Had some problem at some time with destroy view and change detection, but i found an other way to resolve it, so yes, this line is no more needed. Thanks for the reminder.

Reply:
done